composer.json

```diff
@@ -34,11 +34,11 @@
         "phpstan/phpstan-phpunit": "^0.12.16",
         "phpstan/phpstan-strict-rules": "^0.12.5",
         "phpunit/phpunit": "^8.5",
-        "psalm/plugin-phpunit": "^0.13",
+        "psalm/plugin-phpunit": "^0.12",
         "roave/security-advisories": "dev-master",
         "thecodingmachine/phpstan-safe-rule": "^1.0",
         "thecodingmachine/phpstan-strict-rules": "^0.12.1",
-        "vimeo/psalm": "^4.3"
+        "vimeo/psalm": "^3.0"
```

Comment:
Why is a lower version required?

Reply:
Cause psalm 4.x supports php >= 7.3. ;-)